src/gap.c

```diff
@@ -1387,6 +1387,20 @@ static Obj FuncUPDATE_STAT(Obj self, Obj name, Obj newStat)
 }
 
 
+static Obj FuncSetAssertionLevel(Obj self, Obj level)
+{
+    RequireNonnegativeSmallInt("SetAssertionLevel", level);
+    STATE(CurrentAssertionLevel) = INT_INTOBJ(level);
+    return 0;
+}
+
+
+static Obj FuncAssertionLevel(Obj self)
+{
+    return INTOBJ_INT(STATE(CurrentAssertionLevel));
```

Comment:
BTW, placing these into `gap.c` was somewhat arbitrary. If somebody has a preference for another location, let me know.